### ISSUE TITLE
Source-button elements accessible for screen readers ?

### ISSUE THREAD

**Issue by bilrousseau** [thomaspark/bootswatch#910]
Hello,
I tried to open the source code popup which reveals when clicking on source-button div, but the screen reader I'm using can't detect it as a button, so I cannot open that popup.

Is there a way to access this popup via keyboard or after setting the screen reader in a certain way ?

Regards.

**Comment by thomaspark** [comment]
Hi @bilrousseau, thanks for reporting this. I will have a fix for you in the next couple of days.

**Comment by thomaspark** [comment]
@bilrousseau, can you check if the update helps with the issue?
